README.md

```diff
@@ -48,14 +48,18 @@ stackdriver-prometheus-sidecar --help
 
 #### Filters
 
-The `--filter` flag allows to provide filters which all series have to pass before being sent to Stackdriver. The flag may be repeated to provide several filters. Filters use the same syntax as the well-known PromQL label matchers, e.g.:
+The `--filterset` flag allows to provide filters which all series have to pass before being sent to Stackdriver. Filters use the same syntax as [Prometheus time series selectors](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-series-selectors), e.g.:
```

Comment:
I recheck Prometheus document again - we should be really specific on the selector we use.

I think from both design and implementation POV, it's reasonable that we only support "instant vector selectors", not "range vector selectors". "time series selectors" include both of them. The rationale for design is that we should start from minimum set of selectors useful for users - and from what we see, most of the users are using "instant vector selectors" only.

Action item here: change from "time series selectors" to "instant vector selectors", and link to https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors .

Note that your implementation is using ParseMetricSelector, which only works with instant vector selector.

Reply:
Changed, though I think 'time series selectors' would be more clear to someone who is not intimately familiar with Prometheus data model.